contrib/graph-gophers/graphql-go/graphql.go

```diff
@@ -25,9 +25,10 @@ import (
 )
 
 const (
-	tagGraphqlField = "graphql.field"
-	tagGraphqlQuery = "graphql.query"
-	tagGraphqlType  = "graphql.type"
+	tagGraphqlField         = "graphql.field"
+	tagGraphqlQuery         = "graphql.query"
+	tagGraphqlType          = "graphql.type"
+	tagGraphqlOperationName = "graphql.operation.name"
```

Comment:
I don't see any reason to create a new group `graphql.operation.*` since the only key we have in this group is `name`. Can we make it `graphql.operation_name`?

Reply:
Sorry Gabriel, I requested him to make this change to stay consistent with how the JS graphql integration does it. (https://github.com/DataDog/dd-trace-js/blob/aafacd0200b618024af4066b0a1475c0f1e4c55c/packages/datadog-plugin-graphql/src/index.js#L241-L247)

https://github.com/DataDog/dd-trace-go/pull/586#discussion_r378493852

Is that ok with you, or would you prefer `operation_name`?

Reply:
Oh, didn't realise that. Sorry about that. Let's stay consistent! Ignore my comment. Resolving.